### ISSUE TITLE
Artifact caching proxy activation

### ISSUE THREAD

**Issue by lemeurherve** [jenkins-infra/helpdesk#3369]

**Comment by kmartens27** [comment]
Hi @lemeurherve this is great and everything makes sense. Thanks for creating this!
The only things I noticed were a couple instances where the word could be plural:

you have two possibilities: (instead of possibility)
and
If you've got any questions or issues, please add them to this issue (instead of question or issue)

And feel free to disregard if that goes against what you had in mind.

Other than my small suggestions, I think the announcement is terrific!

**Comment by jmMeessen** [comment]
Some nit picking 😉 
I wonder if a casual form like "we've" is appropriate in a rather formal communication. I would have used the form "we have". But I let the word to english native speaker

**Comment by jmMeessen** [comment]
In the same vein, I wonder whether the "mail"/text shouldn't start with a line telling what is the problem we are trying to solve. And than explain the solution. For a blog post that would be important.

**Comment by kmartens27** [comment]
I agree with @jmMeessen on the "we have", for an official announcement as opposed to something more casual. However, depending on the mailing list/forum both can work. There's definitely an explanation of goals/what the solution is within the second line, but having the reason "why" would provide context. The links definitely provide further context to all of the background and reasoning, but if someone decides not to read all of those, the extra info would be helpful.

**Comment by lemeurherve** [comment]
As there were no objection, the PR has been merged, the artifact caching proxy is enabled for every plugin.

**Comment by jglick** [comment]
BTW are there separate tasks to enable it for `jenkins`, `bom`, etc.?

**Comment by dduportal** [comment]
> BTW are there separate tasks to enable it for `jenkins`, `bom`, etc.?

Not yet, we are drafting the plan for it. Most probably we'll continue with the `bom`.

**Comment by lemeurherve** [comment]
Working as expected, closing this issue.
